nav2-usability/scenario-code/deeplink-pathparam/deeplink_pathparam_vrouter.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright 2021, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Deeplink path parameters example
+/// Done using VRouter
+
+import 'package:flutter/material.dart';
+import 'package:vrouter/vrouter.dart';
+
+void main() {
+  runApp(BooksApp());
+}
+
+class Book {
+  final String title;
+  final String author;
+
+  Book(this.title, this.author);
+}
+
+class BooksApp extends StatelessWidget {
+  final List<Book> books = [
+    Book('Stranger in a Strange Land', 'Robert A. Heinlein'),
+    Book('Foundation', 'Isaac Asimov'),
+    Book('Fahrenheit 451', 'Ray Bradbury'),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return VRouter(
+      routes: [
+        VWidget(
+          path: '/',
+          widget: BooksListScreen(books: books),
+          stackedRoutes: [
+            VGuard(
+              beforeEnter: (vRedirector) => validBookId(vRedirector),
+              beforeUpdate: (vRedirector) => validBookId(vRedirector),
```

Comment:
Could you remove the `VGuard` widget? That way it's easier to distinguish this sample from the sign_in_routing (validation) scenario.

Reply:
Done in https://github.com/flutter/uxr/pull/28/commits/02fb450fd3d4774214dae457ae76e472b19fba85